Formula/blockdiag.rb

```diff
@@ -0,0 +1,65 @@
+class Blockdiag < Formula
+  include Language::Python::Virtualenv
+
+  desc "Generate block-diagram images from .diag files"
+  homepage "http://blockdiag.com/en/"
+  url "https://files.pythonhosted.org/packages/b6/1b/eab880ae3e6c7e24ae5adecc40ee7dd05c9ebb0dff6c39b52472844ce9b7/blockdiag-2.0.1.tar.gz"
+  sha256 "16a69dd9f3b44c9e0869999ce82aa968586698febc86ece9ca0c902dba772397"
+
+  depends_on "freetype"
+  depends_on "imagemagick"
+  depends_on "jpeg"
+  depends_on "libtiff"
+  depends_on "little-cms2"
+  depends_on "python"
+  depends_on "webp"
+  depends_on "zlib"
```

Comment:
`brew audit --strict blockdiag` accepts depending on zlib, but `brew audit --new-formula blockdiag` requires that I use `uses_from_macos`, however this does not build if I do not depend on zlib.

Reply:
Why did this `depends_on "zlib"` work on my Mac, but fail on the CI builds?

Reply:
it failed on `brew audit`, we can certainly use `uses_from_macos` one. Let me try it.